.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+### Expected behavior
+
+### Actual behavior
+
+### Steps to reproduce
+
+*Please use markdown syntax, especially for Backtraces and such!*
+
+### program versions
+
+    - operating system
+    - neomutt
```

Comment:
shouldnt' we tell the user to run and give output of `neomutt -v`, and maybe as well of `neomutt -DS`?